src/Folder.js

```diff
@@ -1,12 +1,31 @@
 import Task from './Task';
 import './Folder.css';
 import {useState} from "react";
+import {collection, query, orderBy} from "firebase/firestore";
+import {useCollectionData} from "react-firebase-hooks/firestore";
+
 
 function Folder(props) {
     // used for determining the status of editing the tasks
     const [editFolder,setEditFolder] = useState(false);
     // used to determine the direction of chevron button and whether tasks are shown
     const [showTasks, setShowTasks] = useState(true);
+    // query for tasks collection
+    // retrieving the list of tasks
+    let tasksQuery = ''
+    if (props.folder.sort === "priority") {
+        tasksQuery = query(collection(props.db, "folders", props.folder.id,"tasks"),
```

Comment:
Don't Repeat Yourself. Each of these calls query(collection(...). Only difference is the value of orderBy.  So, just set orderBy in these if/elseif, and then call query afterwards with the calculated orderBy